Sources/AppStoreConnectCLI/Services/Operations/ListAppsOperation.swift

```diff
@@ -32,7 +21,27 @@ struct ListAppsOperation: APIOperation {
     typealias App = AppStoreConnect_Swift_SDK.App
 
     func execute(with requestor: EndpointRequestor) -> AnyPublisher<[App], Error> {
-        requestor.request(options.endpoint).map(\.data).eraseToAnyPublisher()
+        var filters: [ListApps.Filter] = []
+
+        options.bundleIds.isEmpty ? () : filters.append(.bundleId(options.bundleIds))
```

Comment:
🤔 Ok, so this is cute.  But I'm not sure I'm a fan of it.  Generally I'd suggest that ternary operators shouldn't be used for side effects.

```suggestion
        if options.bundleIds.isNotEmpty { filters.append(.bundleId(options.bundleIds)) }
```
This would require:
```
extension Collection {
    var isNotEmpty: Bool { isEmpty == false }
}
``` 
to be defined somewhere in the project. Or alternatively:
```suggestion
        if !options.bundleIds.isEmpty { filters.append(.bundleId(options.bundleIds)) }
```

But we tend to want to avoid the `!` not operator.